client/my-sites/checkout/src/components/payment-method-step.tsx

```diff
@@ -53,19 +53,20 @@ const TotalPrice = styled.div`
 	padding: 16px 0;
 `;
 
-export default function PaymentMethodStep() {
+export default function BeforeSubmitCheckoutHeader() {
 	const cartKey = useCartKey();
 	const { responseCart } = useShoppingCart( cartKey );
 	const taxLineItems = getTaxBreakdownLineItemsFromCart( responseCart );
 	const creditsLineItem = getCreditsLineItemFromCart( responseCart );
+	const shouldCollapseLastStep = useShouldCollapseLastStep();
 	return (
 		<>
-			<CheckoutTermsWrapper>
+			<CheckoutTermsWrapper shouldCollapseLastStep={ shouldCollapseLastStep }>
 				<CheckoutTerms cart={ responseCart } />
 			</CheckoutTermsWrapper>
 
 			{ ! hasCheckoutVersion( '2' ) && (
```

Comment:
Should this be changed to `hasCheckoutVersion( 'collapse-steps')`? Unless the idea is to show this change under regular checkout and _not_ under the collapse-steps flag?

Reply:
Yup, that's the idea. They're separate UI changes.

Reply:
I restored WPOrderReviewSection in 763a322a55c. I hadn't meant to remove it.